app/src/main/java/com/prangesoftwaresolutions/audioanchor/MediaPlayerService.java

```diff
@@ -436,7 +438,6 @@ public void onStop() {
                 //Stop the service
                 stopSelf();
             }
-
```

Comment:
Keep the newline here. In general, try not to make any changes - even small ones like adding a newline or an unneeded import - if they are not part of the feature you are implementing and do not improve the code.